packages/web-react/src/components/ActionGroup/demo/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import DocsSection from '../../../../docs/DocsSections';
+import CustomHorizontalSpacing from './CustomHorizontalSpacing';
+import CustomVerticalSpacing from './CustomVerticalSpacing';
+import DefaultLayout from './Default';
+import HorizontalAlignment from './HorizontalAlignment';
+import HorizontalLayout from './HorizontalLayout';
+import HorizontalReverseLayout from './HorizontalReverseLayout';
+import ResponsiveCustomHorizontalSpacing from './ResponsiveCustomHorizontalSpacing';
+import ResponsiveCustomVerticalSpacing from './ResponsiveCustomVerticalSpacing';
+import ResponsiveHorizontalAlignment from './ResponsiveHorizontalAlignment';
+import ResponsiveHorizontalReverseLayout from './ResponsiveHorizontalReverseLayout';
+import VerticalLayout from './VerticalLayout';
+
+ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
+  <React.StrictMode>
+    <DocsSection title="Basic, Horizontal → Vertical Layout" stackAlignment="stretch">
+      <DefaultLayout />
+    </DocsSection>
+    <DocsSection title="Horizontal Layout" stackAlignment="stretch">
+      <HorizontalLayout />
+    </DocsSection>
+    <DocsSection title="Horizontal Reverse Layout" stackAlignment="stretch">
+      <HorizontalReverseLayout />
```

Comment:
As agreed in person (thanks for pointing it out!), please use the "-ed" form of "reversed" everywhere: in CSS, props and docs.

```suggestion
    <DocsSection title="Horizontal Reversed Layout" stackAlignment="stretch">
      <HorizontalReversedLayout />
```